config/routes.rb

```diff
@@ -12,6 +12,7 @@
         controllers tokens: 'tokens'
         skip_controllers :authorizations, :applications, :authorized_applications, :token_info, :tokens, :confirmations
       end
+      resources :downloads, only: [:index]
```

Comment:
Just for information, not request to change, normally we can define `download` as the additional route of a specific resource.

```ruby
  resources :keywords do
    member do
      get 'download'
    end
  end
```

Reply:
Thanks for your info